libs/frontend/feature-build/src/components/component-menu/ComponentChildStyles.vue

```diff
@@ -37,20 +61,21 @@
         :key="`${entry.pseudoClass}-${entry.property}-${entry.value}`"
         :style="entry"
         :editing="editing(entry.property)"
+        :inheritedFrom="getInheritedFrom(entry)"
```

Comment:
Add the inherited from here so that it shows on overridden styles as well. The style inheritance in overridden styles is working already; it's just that the UI was not showing where is the style entry inherited from.

Reply:
I might be missing something, but is it actually possible to have overridden child styles? Currently we can't assign mixins to children, though in theory it could be implemented.

Reply:
I think so! It's just that styles from larger breakpoints would be the only source of inherited styles for child styles.

https://github.com/samatechtw/pubstudio-builder/assets/59676941/81e07697-1e9f-4e61-9d7f-0923b9ee50f3

Reply:
Oh, that's right!